tools/schematics/dt-component-example/example-tools.utils.ts

```diff
@@ -14,7 +14,10 @@
  * limitations under the License.
  */
 
-import { Rule, Tree } from '@angular-devkit/schematics';
+import {
+  Rule,
+  Tree,
+} from '@nrwl/workspace/node_modules/@angular-devkit/schematics';
```

Comment:
never import from node_modules revert this change

Reply:
I removed all sub-node_module imports.